.github/workflows/build.yaml

```diff
@@ -0,0 +1,69 @@
+name: Build
```

Comment:
do we want create a publish workflow to upload to npm?

Reply:
yea, I was going to go ahead and get this building and then get that automated as well